src/devices/Common/Iot/Device/Common/NumberHelper.cs

```diff
@@ -8,7 +8,12 @@ namespace Iot.Device.Common
     /// <summary>
     /// Helpers for number.
     /// </summary>
-    internal static class NumberHelper
+#if BUILDING_IOT_DEVICE_BINDINGS
+    internal
+#else
+    public
+#endif
+    static class NumberHelper
```

Comment:
another option for ifdefs is do following:
define just `partial class` without any visibility

create a single file which defines all classes and call it like something like: Visibilty.Public.cs, and put just:

```csharp
public partial static class NumberHelper {}
//... other classes
```

and another one with all internal and change csproj to include one or the other

Reply:
This is a good idea. I'll consider it.

Reply:
That worked quite well. Since the default when no visibility is given is "internal", I only had to provide a "Visibility.Public" file.